modules/core/test/test_logtagconfigparser.cpp

```diff
@@ -0,0 +1,281 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "test_precomp.hpp"
+#include <opencv2/core/utils/logger.hpp>
+#include "../src/utils/logtagmanager.hpp"
+#include "../src/utils/logtagconfigparser.hpp"
+
+// Because "LogTagConfigParser" isn't exported from "opencv_core", the only way
+// to perform test is to compile the source code into "opencv_test_core".
+// This workaround may cause step debugger breakpoints to work unreliably.
+#if 1 
```

Comment:
Trailing whitespace will be removed on next commit.
```
modules/core/test/test_logtagconfigparser.cpp:13: trailing whitespace.
+#if 1 
```

Reply:
Consider enabling of commit hook to avoid that: https://github.com/opencv/opencv/wiki/Linux_contribution_guide#git-configuration (step 2)

Reply:
Thanks, I forgot to do that after I set up the clone from scratch.